identify/extensions.py

```diff
@@ -249,6 +249,7 @@
     'swiftdeps': {'text', 'swiftdeps'},
     'tac': {'text', 'twisted', 'python'},
     'tar': {'binary', 'tar'},
+    'templ': {'text', 'go', 'templ'},
```

Comment:
I'm not sure the `go` tag is appropriate here -- the source doesn't look like go syntax? would tools that expect go files be able to accept templ files?

Reply:
Ah no, you're right. I misunderstood the format :) I'll remove `go`